internal/jimmhttp/httpproxy_handler_test.go

```diff
@@ -112,18 +112,32 @@ func (s *httpProxySuite) TestHTTPProxyHandler(c *gc.C) {
 			statusExpected: http.StatusOK,
 			bodyExpected:   "OK",
 		},
+		{
+			description: "invalid model UUID",
+			setup: func() {
+				newURL, _ := url.Parse(fakeController.URL)
+				controller.PublicAddress = newURL.Host
+				err := s.JIMM.Database.UpdateController(ctx, &controller)
+				c.Assert(err, gc.IsNil)
+			},
+			url:            fmt.Sprintf("/model/%s/charms", "fake-uuid"),
+			modelUUID:      "fake-uuid",
+			statusExpected: http.StatusBadRequest,
+			bodyExpected:   "Bad Request - invalid model UUID format",
+		},
 		{
 			description: "model not existing",
 			setup: func() {
 			},
 			url:            fmt.Sprintf("/model/%s/charms", "54d9f921-c45a-4825-8253-74e7edc28066"),
 			modelUUID:      "54d9f921-c45a-4825-8253-74e7edc28066",
 			statusExpected: http.StatusNotFound,
-			bodyExpected:   "Not Found - model not found",
+			bodyExpected:   "Not Found - migrating model .* not found",
 		},
 	}
 
 	for _, test := range tests {
+		c.Log(test.description)
```

Comment:
Do we need this?

Reply:
Yes, when we use gocheck and you run subtests there is no c.Run() like there is with the standard library testing.T. So logging the subtest description is the best way to see which subtest failed. You'll see this often in Juju too.